src/modules/tableOutput.js

```diff
@@ -0,0 +1,18 @@
+const base = document.querySelector('.tableBody');
+
+const retrieve = (item) => {
+  for (let i = 0; i < item.result.length; i++) {
+    const p = document.createElement('p');
+    p.innerHTML = `${item.result[i].user} ${item.result[i].score}`;
+    base.appendChild(p);
+  }
+};
+
+fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/1000/scores/')
+  .then((response) => response.json())
+  .then((json) => retrieve(json))
+  .catch((err) => console.log(err));
+
+document.getElementById('refreshButton').addEventListener('click', () => {
+  location.reload();
+});
```

Comment:
Please make sure to follow the project requirements:
![image](https://user-images.githubusercontent.com/42799579/196829432-67c76e63-1fce-4d8d-bcd8-c27e906ab3c6.png)
Your refresh button should GET data from the API and then you should populate the information that you received.

Your code is displaying the correct information in the table, so what is wrong? I will explain:
Currently, you are fetching data from your API every time the page is loaded (notice that the fetch isn't inside any function and is neither being called anywhere else). This means that your refresh button is just a refresh page button, and isn't getting data from anywhere.
Kindly adjust this, by changing the button functionality to GET the data from the API when clicked, instead of passively refreshing the page.

My tip for you:
You can make an async function to fetch the data and when the button is clicked the function is called. Something like this:
const getData = async () => {
const response = await fetch( ... )
_do stuff with the response, maybe populate the data_
}
Good luck, hope you the best! 😉